meeteval/wer/__main__.py

```diff
@@ -58,12 +58,12 @@ def _load(path: Path):
             raise NotImplemented(f'Unknown file ext: {path.suffix}')
 
 
-def _load_reference(reference: Path):
+def _load_reference(reference: [Path, str, io.TextIOBase, tuple, list]):
```

Comment:
Change to `'Path | str | io.TextIOBase | tuple | list'`. I think that is easier to read and will be the future style (The quotes allow it to be used in python version below 3.10).